packages/frontend/validation-schemas/post/create-auction.ts

```diff
@@ -0,0 +1,71 @@
+import { MIN_DATE } from '@components/primitives/input/date-input';
+import Joi from 'joi';
+import type { TFunction } from 'next-i18next';
+
+export const createAuctionSchema = (t: TFunction) =>
+  Joi.object({
+    category: Joi.string()
+      .required()
+      .messages({
+        'string.empty': t('create-post:validation.category.empty'),
+      }),
+    currency: Joi.any().allow(''),
+    recommendedPriceCurrency: Joi.any().allow(''),
+    minimalBidCurrency: Joi.any().allow(''),
+    callingCode: Joi.any().allow(''),
+    site: Joi.any().allow(''),
```

Comment:
in QA rules we have type URL, so i recommend add something like that
facebook: Joi.string()
      .allow('')
      .uri()
      .max(ValidationRanges.MAX_SOCIAL_NETWORK_URI_SYMBOLS)
      .messages({
        'string.uri': 'error message',
        'string.max': 'error message',
      }),